app/models/patient.rb

```diff
@@ -723,9 +723,8 @@ def latest_transfer
 
     # If a patient record has been inactive for 30 days or more
     # in the exposure workflow and is non-reporting
-    no_recent_activity = where(isolation: false)
-                         .non_reporting
-                         .where('updated_at <= ?', 30.days.ago)
+    no_recent_activity = isolation_non_reporting.where('updated_at <= ?', 30.days.ago)
+                           .or(exposure_non_reporting.where('updated_at <= ?', 30.days.ago))
```

Comment:
Should remove "in the exposure workflow" from comment. The comment above the `:close_eligible` scope should be updated accordingly as well.

Also there's a rubocop alignment issue here.